scripts/modules/MsIndustryLinks/templates/ingest/customconnector/flow_customconnector.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "$connections": {
+        "defaultValue": {},
+        "type": "Object"
+      },
+      "$authentication": {
+        "defaultValue": {},
+        "type": "SecureObject"
+      }
+    },
+    "triggers": {
+      "manual": {
+        "type": "Request",
+        "kind": "Button",
+        "inputs": {
+          "schema": {
+            "type": "object",
+            "properties": {
+              "payload": {
+                "title": "Payload",
+                "type": "string",
+                "x-ms-dynamically-added": true,
+                "description": "The array of items to ingest into Dataverse.",
+                "x-ms-content-hint": "TEXT"
+              }
+            },
+            "required": ["payload"]
+          }
+        }
+      }
+    },
+    "actions": {
+      "Map_data": {
+        "type": "Select",
+        "inputs": {
+          "from": "@array(json(triggerBody()['payload']))",
+          "select": {}
+        }
+      },
+      "Ingest_records": {
```

Comment:
This is basically the same definition as "Ingest_record" below. Can you just reuse this and wrap it with a foreach action instead?

Reply:
Can do. Was tossing up if this was more ideal than how it is currently. Both require significant configuration. Happy to make the change.